### ISSUE TITLE
An issue

### ISSUE THREAD

**Issue by Geervan** [kevinahn7/youtube-clone#5]
When ever I run the app the home page displays fine but when I search something  the search goes on forever how do I solve it 

Thank you

**Comment by kevinahn7** [comment]
This is probably an issue with your API key from YouTube.
Try using a new or updated one

**Comment by Geervan** [comment]
I created another project and used its api but it still isnt working  

This is the log when I use npm start 









Compiled with warnings.

./src/components/Comment.js
  Line 102:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

./src/components/SideList.js
  Line 227:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 228:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 229:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 230:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 231:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 232:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 233:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 234:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 235:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 238:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

./src/methods/convertDate.js
  Line 3:  Missing radix parameter  radix

./src/components/Watch.js
  Line 64:   Missing radix parameter                                                                                    radix
  Line 64:   Missing radix parameter                                                                                    radix
  Line 64:   Missing radix parameter                                                                                    radix
  Line 332:  Missing radix parameter                                                                                    radix
  Line 334:  Missing radix parameter                                                                                    radix
  Line 368:  <iframe> elements must have a unique title property                                                        jsx-a11y/iframe-has-title
  Line 397:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 418:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  Line 443:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

./src/components/VideoThumbnail.js
  Line 64:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
./src/components/ChannelThumbnail.js
  Line 68:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
./src/components/TopNav.js
  Line 86:  Duplicate key 'boxSizing'  no-dupe-keys

./src/components/Results.js
  Line 97:   Expected to return a value at the end of function                                                          array-callback-return
  Line 122:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.
















This is the log of errors when I ran it in localhost
